src/core_plugins/metric_vis/public/_metric_vis.scss

```diff
@@ -0,0 +1,32 @@
+.mtrVis {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.mtrVis__value {
+  font-weight: $euiFontWeightBold;
+  @include euiTextTruncate;
```

Comment:
Small thing. We've normally put the mixins above the properties in the sass.

Reply:
We do? Haha, I didn't notice that pattern.

Reply:
Only reason I do it is because we sometimes overwrite a mixin with properties within the selector (happens in forms a lot) so that way I know what I'm looking at in the selector takes priority over the mixin. Thanks for fixin' um.